antora-playbook.yml

```diff
@@ -60,6 +60,7 @@ antora:
     - require: ./lib/antora/versions-sorter-extension.js
     - require: ./lib/antora/versions-alias-extension.js
     - require: '@antora/atlas-extension'
+    - require: '@asciidoctor/tabs'
```

Comment:
⚠️ this seems to be an asciidoctor extension, so it shouldn't be placed here which is intended for Antora extensions: https://docs.antora.org/antora/latest/extend/register-extension/
For Asciidoctor extension, see https://docs.antora.org/antora/latest/playbook/asciidoc-extensions/

Reply:
Yep my mistake it is under the wrong key